packages/server/lib/db/migrations/20230510083713_activity_logging_table.cjs

```diff
@@ -0,0 +1,50 @@
+const tableName = '_nango_activity_logs';
+const messagesTableName = '_nango_activity_log_messages';
+
+exports.up = function (knex, _) {
+    return knex.schema
+        .withSchema('nango')
+        .createTable(tableName, function (table) {
+            table.increments('id').primary();
+            table.integer('account_id').unsigned().notNullable();
+            table.enu('level', ['error', 'warn', 'info', 'http', 'verbose', 'debug', 'silly']).defaultTo('info').notNullable();
+            table.enu('action', ['oauth', 'proxy', 'token', 'sync']).defaultTo('oauth').notNullable();
+            table.boolean('success');
+            table.bigInteger('timestamp');
+            table.bigInteger('start');
+            table.bigInteger('end');
+            table.text('endpoint');
+            table.string('provider_config_key');
+            table.string('connection_id');
+            table.string('provider');
+            table.enum('method', ['GET', 'POST', 'PUT', 'PATCH', 'DELETE']);
+            table.string('session_id');
+
+            table.foreign('account_id').references('id').inTable('nango._nango_accounts').onDelete('CASCADE');
```

Comment:
Also, I wonder if it makes to index `account_id` here since we are querying by this?

Reply:
Great point! Added with https://github.com/NangoHQ/nango/pull/633/commits/43773f656c85e02b14f9de6c9df497eec2931be0